ui/src/adapters/api/credentials.ts

```diff
@@ -108,15 +115,20 @@ export async function getCredentials({
         ...(did !== undefined ? { did } : {}),
         ...(query !== undefined ? { [QUERY_SEARCH_PARAM]: query } : {}),
         ...(status !== undefined && status !== "all" ? { [STATUS_SEARCH_PARAM]: status } : {}),
+        ...(max_results !== undefined ? { max_results: max_results.toString() } : {}),
+        ...(page !== undefined ? { page: page.toString() } : {}),
       }),
       signal,
       url: `${API_VERSION}/credentials`,
     });
     return buildSuccessResponse(
-      getListParser(credentialParser)
-        .transform(({ failed, successful }) => ({
-          failed,
-          successful: successful.sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime()),
+      getResourceParser(credentialParser)
+        .transform(({ items: { failed, successful }, meta }) => ({
+          items: {
+            failed,
+            successful: successful.sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime()),
```

Comment:
I'd remove this sorting by the moment as we agreed.

Reply:
Done